oaknational/Oak-Web-Application
# [CUR-326] Feat/unit modal navigates to unit

## Description

Music year: 1970

- Removed coming soon tag on unit modal when lesson is available

## How to test

1. Go to https://deploy-preview-2142--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/units
2. Click on Simple Sentences
3. You should see the button to take you to the unit. This should redirect to the correct unit listing page
4. Unit optionality modals should have no button or tag
5. Remaining unit modals should have a coming soon label alongside the unit info button

## Screenshots

How it used to look (delete if n/a):
<img width="697" alt="Screenshot 2023-12-21 at 12 09 16" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/9dfc1eda-583d-4375-870f-f86590a04c10">

How it should now look:
<img width="736" alt="Screenshot 2023-12-21 at 12 08 22" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/5194f816-b952-4741-898d-f843ae2d9560">

## Checklist
